test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -61,4 +61,5 @@ jest.mock('@folio/stripes/components', () => ({
   Select: jest.fn(() => <select> </select>),
   TextArea: jest.fn((props) => <textarea {...props} />),
   TextField: jest.fn((props) => <input {...props} />),
+  Datepicker: jest.fn((props) => <div {...props} />),
```

Comment:
Please add components in alphabetical order for easier usage

Reply:
done